.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: python
+dist: xenial
```

Comment:
bionic beaver is the most up to date LTS version

Reply:
Yes but I think travis doesn't offer that!